README.md

```diff
@@ -32,3 +54,34 @@ var v3 = new Vector(3, 4)
 console.log(v3.getLength());
 // => 5
 ```
+
+
+class Vector{
+    constructor(x,y){
+      this.x = x;
+      this.y = y;
+    }
+    plus(num1){
+      let x = this.x + num1.x;
+      let y = this.y + num1.y;
+      let v3 = new Vector (x,y)
+      return v3
+    }
+    minus(num1){
+      let x = this.x - num1.x;
+      let y = this.y - num1.y;
+      let v4 = new Vector (x,y)
+      return v4
+    }
+    getLength(){
+      return Math.sqrt((this.x * this.x) + (this.y * this.y))
+    }
+}
+
```

Comment:
Solid job, but the argument names could be clearer. Argument and variable names should be somewhat clear as to what they are. If I was reading through and saw the argument "num1" then I would expect a number and not a Vector/object.